### ISSUE TITLE
No module named 'tronapi.providers'

### ISSUE THREAD

**Issue by fffy2366** [iexbase/tron-api-python#19]
Traceback (most recent call last):
  File "alert.py", line 1, in <module>
    from tronapi import Tron
  File "/usr/local/lib/python3.6/site-packages/tronapi/__init__.py", line 12, in <module>
    from tronapi.providers.http import HttpProvider  # noqa: E402
ModuleNotFoundError: No module named 'tronapi.providers'

**Comment by serderovsh** [comment]
tronapi.providers - not needed

```python
# current
from tronapi import Tron 
tron = Tron()

account = tron.create_account
is_valid = bool(tron.isAddress(account.address.hex))

logger.debug('Generated account: ')
logger.debug('- Private Key: ' + account.private_key)
logger.debug('- Public Key: ' + account.public_key)
logger.debug('- Address: ')
logger.debug('-- Base58: ' + account.address.base58)
logger.debug('-- Hex: ' + account.address.hex)
logger.debug('-- isValid: ' + str(is_valid))
logger.debug('-----------')
```

**Comment by serderovsh** [comment]
Example: https://github.com/iexbase/tron-api-python/blob/master/examples/account.py

**Comment by serderovsh** [comment]
Turns out there was a problem.
Issue resolved, upgrade to 3.0.2 [#311](https://github.com/iexbase/tron-api-python/commit/fd3d36aca61d7cc6c265b9128c46beefc9deaccf)

Thanks!

**Comment by qq29oo** [comment]
The version 3.1.0 show this problem again.

Traceback (most recent call last):
  File "tron_deploy.py", line 1, in <module>
    from tronapi import Tron, HttpProvider
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/tronapi-3.1.0-py3.7.egg/tronapi/__init__.py", line 12, in <module>
    from tronapi.providers.http import HttpProvider  # noqa: E402
ModuleNotFoundError: No module named 'tronapi.providers'
